app/safe/internal/bootstrap/k8s.go

```diff
@@ -13,23 +13,22 @@ package bootstrap
 import (
 	"context"
 	"github.com/pkg/errors"
-	"github.com/vmware-tanzu/secrets-manager/core/log"
 	"github.com/vmware-tanzu/secrets-manager/app/safe/internal/state"
 	"github.com/vmware-tanzu/secrets-manager/core/env"
+	"github.com/vmware-tanzu/secrets-manager/core/log"
 	v1 "k8s.io/api/core/v1"
 	metaV1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/client-go/kubernetes"
 	"k8s.io/client-go/rest"
 )
 
 var (
-	vsecmSystemNamespace string
 	id string = "VSECMSAFE"
 )
 
-func init() {
+func ValidateEnvironment() {
 	// getting metadata.namespace, passed through environment variable VSECM_SYSTEM_NAMESPACE
-	if vsecmSystemNamespace = env.InitVsecmSystemNamespace(); len(vsecmSystemNamespace) == 0 {
+	if len(env.SystemNamespace()) == 0 {
```

Comment:
The `Init` there is misleading we are not initing anything.

I think it was confused with the `Init` of the `InitContainer` while following the pattern in the file.

An “init container” (as we all know), is a Kubernetes primitive.

that’s why the variable was named `InitContainerPollInterval`: it is the poll interval of the init container. 

it is not initializing the poll interval of anything, for example.